internal/connectors/plugins/public/adyen/plugin_test.go

```diff
@@ -0,0 +1,135 @@
+package adyen
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/formancehq/payments/internal/connectors/plugins"
+	"github.com/formancehq/payments/internal/connectors/plugins/public/adyen/client"
+	"github.com/formancehq/payments/internal/models"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	gomock "go.uber.org/mock/gomock"
+)
+
+func TestPlugin(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Stripe Plugin Suite")
+}
+
+var _ = Describe("Stripe Plugin", func() {
```

Comment:
👀 this test suite shouldn't be called "Stripe"

Reply:
Oopsie ! Fixed thanks